src/app/containers/Image/__snapshots__/index.test.jsx.snap

```diff
@@ -57,5 +62,6 @@ exports[`Image with data should render an image with the default originCode \`cp
   copyright="Getty images"
   ratio={65.625}
   src="https://ichef.bbci.co.uk/news/640/cpsdevpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png"
+  srcSet="https://ichef.bbci.co.uk/news/360/cpsdevpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png 360w, https://ichef.bbci.co.uk/news/480/cpsdevpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png 480w, https://ichef.bbci.co.uk/news/600/cpsdevpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png 600w, https://ichef.bbci.co.uk/news/720/cpsdevpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png 720w, https://ichef.bbci.co.uk/news/840/cpsdevpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png 840w, https://ichef.bbci.co.uk/news/1024/cpsdevpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png 1024w, https://ichef.bbci.co.uk/news/2048/cpsdevpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png 2048w"
```

Comment:
Should these be valid URLs? I get 403s on them

Reply:
These seem to be 403'ing due to us currently hardcoding to `cpsdevpb` while be integrate with Optimo Test, in the future they will be dictated by the originCode in the feed